### DISCUSSION TITLE
Is `b` a defined symbol? (a=b are allowed, but a=c is a undefined symbol)

### DISCUSSION THREAD

**Discussion by exetico** [josdejong/mathjs#3293] [Q&A] [answered]
Hi,

I just wondered why `a=b` are handled well without a error thrown from mathjs. Where a=c complains about `c` are a undefined symbol.

But; why are b a defined symbol in the first place?

Here's a simple recreation from Math Nodepad:
<img width="328" alt="billede" src="https://github.com/user-attachments/assets/43752295-cd21-419d-bbcf-6aa9fe4f23cd">

I noticed this while integrated Mathjs into a solution, and wondered why `a=b` is allowed in the first place.

**Comment by josdejong** [comment, marked as answer]
The `b` is the unit `bits`, see https://mathjs.org/docs/datatypes/units.html#reference

**Reply by exetico** [reply]
Oh, I see. Thanks. I actually checked the units page, but must have been overlooking the Binary > bits. Thanks!
